In [14]:
from math import factorial as fcl
import numpy as np
from scipy.stats import norm

### Задание 1.

Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

#### Решение:

Для равномерного распределения:
$$M(x) = \frac{a+b}{2}$$  
$$D = \frac{(b-a)^2}{12}$$

In [1]:
M = (200 + 800)/2
M

500.0

In [3]:
D = (800-200)**2/12
D

30000.0

### Задание 2.

О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

#### Решение:

$$D = \frac{(b-a)^2}{12}$$

$0.2 = \frac{(b-0.5)^2}{12} \Rightarrow 0.2 \cdot 12 = (b-0.5)^2 \Rightarrow b-0.5 = \sqrt{2.4} \Rightarrow \pm 1.549+0.5 \Rightarrow 1.549+0.5=2.049$

In [7]:
np.sqrt(0.2*12)

1.5491933384829668

$$M(x) = \frac{a+b}{2}$$  

In [11]:
M = (0.5 + 2.049)/2
M

1.2745

### Задание 3.

Непрерывная случайная величина X распределена нормально и задана плотностью распределения  

$$f(x) = (1 / (4 * sqrt(2pi))) * exp((-(x+2)**2) / 32)$$  

Найдите:  
а). M(X)  
б). D(X)  
в). std(X) (среднее квадратичное отклонение)

#### Решение:

<font size="6"> 
$f(x) = \frac{1}{\sigma\sqrt{2\pi}}e^{-\frac{(x-\mu)^2}{2\sigma^2}}$   
    
$f(x) = \frac{1}{4 \cdot \sqrt{2\pi}} \cdot e^{-\frac{(x+2)^2}{32}}$

$M(X) = -2$  
$D(X) = 16$  
$\sigma(X) = 4$

### Задание 4.

Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:  

а). больше 182 см  
б). больше 190 см  
в). от 166 см до 190 см  
г). от 166 см до 182 см  
д). от 158 см до 190 см  
е). не выше 150 см или не ниже 190 см  
ё). не выше 150 см или не ниже 198 см  
ж). ниже 166 см.

#### Решение:

а). больше 182 см 

In [16]:
1-norm.cdf(1)

0.15865525393145707

б). больше 190 см  

In [17]:
1-norm.cdf(2)

0.02275013194817921

в). от 166 см до 190 см 

In [21]:
norm.cdf(2) - norm.cdf(-1)

0.8185946141203637

г). от 166 см до 182 см

In [22]:
norm.cdf(1) - norm.cdf(-1)

0.6826894921370859

д). от 158 см до 190 см  

In [23]:
norm.cdf(2) - norm.cdf(-2)

0.9544997361036416

е). не выше 150 см или не ниже 190 см

In [24]:
norm.cdf(-3) + (1-norm.cdf(2))

0.0241000299798093

ё). не выше 150 см или не ниже 198 см

In [25]:
norm.cdf(-3) + (1-norm.cdf(3))

0.0026997960632601965

ж). ниже 166 см.

In [26]:
norm.cdf(-1)

0.15865525393145707

#### Ответ:

а). 16%  
б). 2.27%  
в). 81.9%  
г). 68.2%  
д). 95.4%  
е). 2.41%  
ё). 0.27%  
ж). 16%

### Задание 5.

 На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

#### Решение:

<font size="4"> 
$$Z = \frac{X-\mu}{\sigma}$$

<font size="4"> 
$Z=\frac{190 - 178}{\sqrt{25}} = \frac{12}{5} = 2.4$

In [13]:
12/5

2.4